src/gui/mod.rs

```diff
@@ -160,6 +166,9 @@ pub struct Ajour {
 impl Default for Ajour {
     fn default() -> Self {
         Self {
+            state: [(Mode::Catalog, State::Loading)].iter().cloned().collect(),
```

Comment:
Initially start catalog in `State::Loading` state as this is the only part we know for sure will be there.
The `MyAddon(flavor)` will be added later.

Reply:
Makes sense!